broker/src/tunneldigger_broker/tunnel.py

```diff
@@ -350,7 +288,7 @@ def create_tunnel(self, address, uuid, remote_tunnel_id, client_features):
         """
 
         if address != self.endpoint:
-            logger.warning("Protocol error: tunnel endpoint has changed.")
+            logger.warning("Protocol error: tunnel endpoint has changedi. Possibly due to kernel bug. See: https://github.com/wlanslovenija/tunneldigger/issues/126")
```

Comment:
```suggestion
            logger.warning("Protocol error: tunnel endpoint has changed. Possibly due to kernel bug. See: https://github.com/wlanslovenija/tunneldigger/issues/126")
```

Reply:
You can clearly tell I use vim :yum: 
Fixed and force-pushed.